custom_addons/softifi_graph_module_dependency/static/description/index.html

```
@@ -0,0 +1,16 @@
+<section class="oe_container">
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)